includes/logs.inc

```diff
@@ -17,21 +17,30 @@ function islandora_find_replace_operation_logs() {
   $header = array(
     t('ID'),
     t('Content Model'),
+    t('Object ID'),
     t('DSID'),
     t('Find'),
     t('Replace'),
   );
   $rows = array();
   foreach ($operations as $operation) {
+    $objectIDs = '';
+    $revisions = unserialize($operation->revisions);
+    if (is_array($revisions) || is_object($revisions)){
+      foreach($revisions['success'] as $key => $value){  
+        $objectIDs = $objectIDs .' <p> ' . $key . '</p>';
+      }
+    }
     $rows[] = array(
       l($operation->id, 'admin/islandora/tools/find-replace/log/' . $operation->id),
       check_plain($operation->cmodel),
+      $objectIDs,
```

Comment:
Does this include a full list of all objects affected? mmm-- sounds a bit too much? Or am I understanding this wrong?

Reply:
Hmm, I don't know about this one - not my addition; that came from the commits I pulled from the other repos. 

The line does seem to be problematic (along with possibly others in the log file). Here's an error I get when I look at the log for an individual operation:

> Notice: Trying to get property of non-object in islandora_find_replace_operation_log() (line 40 of /var/www/drupal/sites/all/modules/islandora_find_replace/includes/log.inc).
> Notice: Trying to get property of non-object in islandora_find_replace_operation_log() (line 40 of /var/www/drupal/sites/all/modules/islandora_find_replace/includes/log.inc).
> Notice: Trying to get property of non-object in islandora_find_replace_operation_log() (line 42 of /var/www/drupal/sites/all/modules/islandora_find_replace/includes/log.inc).

Reply:
I think this is good. It gives you a full list of objects, but only after you've clicked into the Replacement ID - which in my opinion is exactly what you want. Lets you check on individual objects that might have been adversely affected by the find/replace operation.

Reply:
Ok, good catch. I still have to give it a run, first review was purely theory =)

Reply:
Ha -- OK, I found the source of the error I mentioned. The error came up because the object in the log no longer exists in the repository (I deleted it at some point after the find/replace operation was carried out). So there should be some sort of check to see if the object exists -- but not sure what to output if it doesn't.

Reply:
Ok, good to know!. Can you open an issue in this repo? Simply put: deal with deleted objects that are being referenced in the logs. =)

Reply:
Issues appear to be disabled in this repo.

Reply:
Ups, sorry, perks of cloning. Now there are active. Also: I will add automatic testing/DCS checks etc. soon (just in case you want to keep on collaborating). Thanks!